merge_values_test.go

```diff
@@ -41,13 +41,134 @@ nested:
   deeper:
     bottom: true
     float: 274877.906944
+test: test`
+	overrideNestedYaml = `
```

Comment:
What about having separate constant blocks for these things? Took me a while to figure what is going on here.

Reply:
Good point that is easier to read. Thanks, done.